easybuild/tools/github.py

```diff
@@ -58,6 +64,12 @@
     _log.warning("Failed to import from 'vsc.utils.rest' Python module: %s" % err)
     HAVE_GITHUB_API = False
 
+try:
+    import git
+    from git import GitCommandError
+except ImportError as err:
+    _log.warning("Failed to import 'git' Python module: %s", err)
```

Comment:
we do, and I'm using it lower down, see `@only_if_module_is_available`

but `git` still needs to be imported, the decorator doesn't take care of that (it maybe could though, but I'll keep that for when I have time to spare once ;-))